system/README.md

```diff
@@ -0,0 +1,22 @@
+## System
+
+This module emit the actual network status.
```

Comment:
What does this mean? Please describe what the System module is.

Reply:
b62de49